ext/process/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018-2025 the Deno authors. MIT license.
+
+[package]
+name = "deno_process"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+readme = "README.md"
+repository.workspace = true
+description = "Subprocess APIs for Deno"
+
+[lib]
+path = "lib.rs"
+
+[dependencies]
+deno_core.workspace = true
+deno_error.workspace = true
+deno_fs.workspace = true
+deno_io.workspace = true
+# TODO(bartlomieju): this crate shouldn't depend on `deno_node`.
+deno_node.workspace = true
```

Comment:
This is an implicit circular dependency. Should move relevant structs from `deno_node` to this extension